### DISCUSSION TITLE
Split diff button not working

### DISCUSSION THREAD

**Discussion by Talon1024** [community/community#42056] [Repositories]
### Select Topic Area

Bug

### Body

If I click the "Split" button to show the split diff instead of the unified diff, it takes me to https://github.com/?diff=split instead of showing the split diff. In order to see the split diff, I have to append `?diff=split` to the URL manually.

**Comment by jfullstackdev** [comment]
I don't need to do that 

![image](https://user-images.githubusercontent.com/47092464/208307596-6bf37c02-c03b-4613-8069-2e9758dec80a.png)

is it the same view that you are seeing ?

**Comment by Talon1024** [comment]
It may not be necessary for small, simple changes like that. But, for [more complex changes such as this](https://github.com/Talon1024/rustmd3view/commit/ff2a251b87c3f2da93a22bd7656c64c0b1d86d52), the split diff view is more helpful.

**Reply by jfullstackdev** [reply]
![image](https://user-images.githubusercontent.com/47092464/210139407-1c96cbdf-caf5-4b5e-9889-2112880cc537.png)

![image](https://user-images.githubusercontent.com/47092464/210139412-d5a201a1-0b51-4316-b7b0-b7eabd166e53.png)

still no problem on my end for the two views

**Comment by FFFluoride** [comment]
It probably only works on Chrome based browser, idk.
On firefox it does the exact same thing. When I use brave it works as intended. 
~~Common microsoft skill issue~~
